interfaces/builtin/hidraw.go

```diff
@@ -113,24 +113,25 @@ func (iface *HidrawInterface) SanitizePlug(plug *interfaces.Plug) error {
 	return nil
 }
 
+func (iface *HidrawInterface) UdevPermanentSlot(spec *udev.Specification, slot *interfaces.Slot) error {
+	usbVendor, vOk := slot.Attrs["usb-vendor"].(int64)
+	if !vOk {
+		return nil
+	}
+	usbProduct, pOk := slot.Attrs["usb-product"].(int64)
+	if !pOk {
+		return nil
+	}
+	path, ok := slot.Attrs["path"].(string)
+	if !ok || path == "" {
+		return nil
+	}
+	spec.AddSnippet(string(udevUsbDeviceSnippet("hidraw", usbVendor, usbProduct, "SYMLINK", strings.TrimPrefix(path, "/dev/"))))
```

Comment:
Ah, yes, in fact I changed udevUsbDeviceSnippet to return string at the end, just forgot to revisit this line. Thanks, fixed now.